corelight/MANIFEST

```diff
@@ -981,12 +981,27 @@
 			"Name": "corelight_x509",
 			"Type": 4,
 			"Hash": "0000000000000000000000000000000000000000000000000000000000000000"
+		},
+		{
+			"Name": "4a6a2042-faaf-4d0a-8fa4-df8a90d3aa65",
```

Comment:
Is this really the name you intended to use?

Reply:
yes, the Name is just the UUID of the dashboard.